torch/_C/__init__.pyi.in

```diff
@@ -409,6 +410,13 @@ class FunctionSchema:
     returns: List[Argument]
     ...
 
+# Defined in torch/csrc/jit/python/script_init.cpp
+class TorchScriptSerializer(object):
```

Comment:
I'll comment on the PR that actually does the binding too, but: I think we should name this the same thing it's named in C++ (`ScriptModuleSerializer`). Seems unnecessarily confusing to change the name when binding the type into Python. Future ppl should be able to grep for the object name and find the cpp implementation without delving into binding code.